cinema-backend/src/main/kotlin/pl/theliver/cinemabackend/presentation/model/SeanceDto.kt

```diff
@@ -0,0 +1,29 @@
+package pl.theliver.cinemabackend.presentation.model
+
+import pl.theliver.cinemabackend.application.services.ScreeningRoomService
+import pl.theliver.cinemabackend.domain.Seance
+import java.util.*
+import java.time.format.DateTimeFormatter
+
+data class SeanceDto(
+        val id: String,
+        val startDate: String,
+        val movieId: String,
+        val screeningRoomId: String,
+        val screeningRoomName: String
+) {
+    companion object {
+        fun fromDomain(seance: Seance,
+                       screeningRoomService: ScreeningRoomService
```

Comment:
I think that it is bad when you need service to make mapping, maybe remove this field from SeanceDto